src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/CanonicalDefinitionEETypeNode.cs

```diff
@@ -17,16 +18,25 @@ public CanonicalDefinitionEETypeNode(NodeFactory factory, TypeDesc type) : base(
             Debug.Assert(type.IsCanonicalDefinitionType(CanonicalFormKind.Any));
         }
 
-        public override bool ShouldSkipEmittingObjectNode(NodeFactory factory) => false;
```

Comment:
Why would it break multimodule? This should fallback to the base type's implementation, which currently will return true only if a constructed version of the type exists.